src/libraries/System.Text.Json/src/System/Text/Json/JsonException.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Text.Json
     public class JsonException : Exception
     {
         // Allow the message to mutate to avoid re-throwing and losing the StackTrace to an inner exception.
-        private string _message;
+        internal string? _message;
```

Comment:
Why is the nullablility flow changing this from private to internal?

Reply:
That is kind of a long story. Exception.Message getter shouldn't be nullable (contract) so we changed the getter from `return _message` into `return _message ?? base.Message;`. But there is a logic in ThrowHelper set the message when `Message` is null set t https://github.com/dotnet/runtime/pull/114/files#diff-e586edfb9caf385bf493d49a97f197a7R217 which not setting correct message anymore after changing `Message` to non null. So instead of using `Message` property  we changed it to use this field direclty `ex._message;` for which the field need to be internal